### PR TITLE
Prague 2021 blog post thank you and recap

### PR DESCRIPTION
**Pull request by thatdocslady**

### CODE REVIEW

File: docs/conf/prague/2021/news/thank-you-recap.rst [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+Thanks again to all our presenters and moderators! Huge thanks also to `Bart at Work`_ for handling the A/V production on our event, it was flawless once again this year.
+
+.. _YouTube: https://www.youtube.com/playlist?list=PLZAeFn6dfHpnaoiOQyd9BYbQbprDGQjQ9
+.. _Bart at Work: https://www.bartatwork.com/
+.. _YouTube channel: https://www.youtube.com/writethedocs
+
+Shirts still on sale
+--------------------
+
+If you want a souvenir from this year's conference, our `Write the Docs Prague 2021 Pop-Up Shop <https://https://shirts.writethedocs.org/>`_ is still open!
```
Comment (thatdocslady) [inline docs/conf/prague/2021/news/thank-you-recap.rst]:
```suggestion
If you want a souvenir from this year's conference, our `Write the Docs Prague 2021 Pop-Up Shop <https://shirts.writethedocs.org/>`_ is still open!
```

File: docs/conf/prague/2021/news/thank-you-recap.rst [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+We have loose and fitted options from different fabrics, and you can choose exactly the size and cut you want. The campaign will run in 3-day batches for another few weeks. Shipping is available from the USA or Europe.
+
+**New this year**: You can shop for non-conference swag in our `Write the Docs Community Shop <https://https://writethedocs.threadless.com/>`_ on Threadless.
```
Comment (thatdocslady) [inline docs/conf/prague/2021/news/thank-you-recap.rst]:
```suggestion
**New this year**: You can shop for non-conference swag in our `Write the Docs Community Shop <https://writethedocs.threadless.com/>`_ on Threadless. 
```

File: docs/conf/prague/2021/news/thank-you-recap.rst [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+Thanks again to all our presenters and moderators! Huge thanks also to `Bart at Work`_ for handling the A/V production on our event, it was flawless once again this year.
+
+.. _YouTube: https://www.youtube.com/playlist?list=PLZAeFn6dfHpnaoiOQyd9BYbQbprDGQjQ9
+.. _Bart at Work: https://www.bartatwork.com/
+.. _YouTube channel: https://www.youtube.com/writethedocs
+
+Shirts still on sale
+--------------------
+
+If you want a souvenir from this year's conference, our `Write the Docs Prague 2021 Pop-Up Shop <https://shirts.writethedocs.org/>`_ is still open!
```
Comment (thatdocslady) [inline docs/conf/prague/2021/news/thank-you-recap.rst]:
```suggestion
If you want a souvenir from this year's conference, our `Write the Docs Prague 2021 Pop-Up Shop <https://shirt.writethedocs.org/>`_ is still open!
```

**Review by ericholscher — APPROVED** [review]
Looks good to me 👍

**Pull request merged**